db/seeds.rb

```diff
@@ -1,5 +1,7 @@
 include InventoryExceptions
 
+dept = Department.create(name: '_test')
+User.create(first_name: 'Apples', last_name: 'Sauce', spire_id: '00000000', email: "applesauce@xyz.net", phone: '1248279421', department: dept)
```

Comment:
xyz.net could actually be a URL, why not just use example.com for everything? :)

Reply:
This was just part of the debugging me and Tom did, I doubt this will stay exactly like this when it's fully fixed.